src/opnsense/mvc/app/controllers/OPNsense/Base/ApiMutableModelControllerBase.php

```diff
@@ -107,22 +107,6 @@ protected function getModel()
     }
 
     /**
-     * validate and save model after update or insertion.
-     * Use the reference node and tag to rename validation output for a specific node to a new offset, which makes
-     * it easier to reference specific uuids without having to use them in the frontend descriptions.
-     * @param $node reference node, to use as relative offset
-     * @return array result / validation output
-     */
-    protected function validateAndSave($node = null)
```

Comment:
I'm sorry, I don't understand your comment. The hooks don't run on save. They run on individual actions. The intention of the hooks is to stop the programmer from having to fully re-implement the boilerplate actions. The hooks are noops by default. Either way I don't understand what this has to do with removing dead code.

If you want to keep it in because it may be useful, why not.